File: android/src/main/java/com/ownfin/aes/encoding/Base64.java
package com.ownfin.aes.encoding;

public class Base64 {

    public static String toString(byte[] inputBytes) {
        return android.util.Base64.encodeToString(inputBytes, android.util.Base64.NO_WRAP);
    }
    public static byte[] toBytes(String inputBase) {
        return android.util.Base64.decode(inputBase, android.util.Base64.NO_WRAP);
    }

}

File: ios/RCTAes/Encoding/Base64.m
//
//  Base64.m
//
//  Created by bastiandev on 19.05.21.
//  Copyright © 2021 ownfin. All rights reserved.
//

#import "Base64.h"

@implementation Base64
+ (NSString *) toString:(NSData *)inputBytes {
    return [inputBytes base64EncodedStringWithOptions:0];
}
+ (NSData *) toBytes: (NSString *)inputBase {
    if(inputBase != nil){
        return [[NSData alloc] initWithBase64EncodedString:inputBase options:0];
    }
    return nil;
}
@end
